index.html

```diff
@@ -398,19 +398,44 @@ <h2 id="syllabus">Syllabus</h2>
                         <h2 id="syllabus">AMA Responses</h2>
                         <p>Thanks for sending all your questions in! We've compiled our responses below. Feel free to reach out to us or ask in class if you have additional questions!</p>
                         <details>
-                            <summary class="font-semibold cursor-pointer">What major(s) should I consider if I'd like to know more about ethics & CS? <span class="badge ~urge">New</span></summary>
+                            <summary class="font-semibold cursor-pointer">What's a good class to take after 106B for someone who's not a CS major but wants to keep some CS skills without spending tooooo much time on homework? <span class="badge ~urge">New</span></summary>
+                            <div class="content mt-2">
+                                <p>Carta's hours-per-week feature is definitely your friend here. :D Here are some of my suggestions:</p>
+                                <ul>
+                                    <li>CS 51 + CS 52! CS Studio is a really fun class and great way to use your CS skills for social good! Definitely more going on than in this class, since you are expected to build something by the end, but you will have a team to support you and there's no letter grade at stake. Applications will probably open in the next couple of weeks -- join <a href="https://bit.ly/cssg-interest-2122">our mailing list</a> to get notified when they open! </li>
+                                    <li>CS 106L, if you're interested in learning C++ standard libraries (that is, what's used in the "real world," not just the Stanford version you see in 106B). </li>
+                                    <li>CS 109 - if you're okay/good with math/probability, 109 has a reputation of being the chillest class in the CS core. At the end you do some really cool machine learning stuff that totally blew me away! That said, it is still a 5 unit class, and the only all-nighter I've ever had to pull was when I severely underestimated how long the last 109 homework would take. </li>
+                                    <li>CS 124, if you're interested in NLP (e.g., if you thought week 3 of this class was interesting!). It's 4 units and generally pretty chill, with the exception of the first assignment (writing a ton of regex) and the final (group) project (building a pretty robust chatbot).</li>
+                                    <li>CS 182, if you're interested in ethics (which I assume you are if you're in this class!) and are okay with writing. I haven't taken this class yet (although it's definitely on my list!), but my friends who have said it is fairly similar to stuff we will be talking about week 6.</li>
```

Comment:
Could we adjust this to be consistent between dashes after the class names and commas?